packages/components/src/Combobox/constants.ts

```diff
@@ -0,0 +1 @@
+export const COMBOBOX_MENU_ID = "ATL-Combobox-Content";
```

Comment:
It may be worth exporting this in `index.ts`. We have done that for Glimmer and Grid but no where else so it isn't major and us exporting Test ids  can be something we discuss as a team

Reply:
fair point. I haven't checked to see if people are referencing this directly in product or not.

also related to your other idea - this should have some sort of unique identifier appended for the multiple combobox case. I suppose the base ID wouldn't change, assuming we do just append something unique anyway.

Reply:
I think so too, we could allow the passing of a "testIDSuffix` for these types of components and append it and export the "base" test ids so they can get it by using the suffix they add. However this is way out of scope and a bigger discussion because there is probably a way better way to do this